src/60_Samples_%26_Templates/Product_Browse_Page.html

```diff
@@ -127,7 +135,7 @@
          .autosuggest-box {
             position: absolute;
             width: 100%;
-            background-color: #fafafa;
+            background-color: var(--color-text-light);
             box-shadow: 0px 2px 6px rgba(0,0,0,.3);
           }
           .hidden {
```

Comment:
Do we still need the hidden rule? Or could we replace this with the attribute?

Reply:
removed